db/db.js

```diff
@@ -379,6 +379,24 @@ async function getLocationData(locationid) {
 
 }
 
+// Retrieves the locationid corresponding to a particle device coreID
+
+async function getLocationFromCoreID(coreID){
```

Comment:
If you're only getting the locationId, then I suggest that your change the name of the function to make that clear `getLocationIdFromCoreID` or something like that.